impeller/entity/shaders/filters/gaussian.frag

```diff
@@ -9,6 +9,8 @@
 
```

Comment:
We should keep the name of this file as `kernel.glsl` since our goal is to minimize the number of shaders and this shader isn't specific to gaussian blur.

Reply:
i'd argue we could pursue that if we have a use for that, but until then YAGNI

Reply:
There's nothing to pursue, it's already in a good state, just undo the changes =)  Had we named our shaders based on what they do versus what they are used for in the past we may not have had to do all this cleanup now.

Reply:
Its only used for a gaussian blur, the name is more correct